packages/odata-v2/src/common.ts

```diff
@@ -0,0 +1,22 @@
+export {
```

Comment:
I like this approach 😊 The question is, if this should be "internal" in `odata-common`

Reply:
We thought about this too, our thought was that it's cleaner if the clients don't have to import from both packages, that's why we re-exported it instead.